requirements.txt

```diff
@@ -12,3 +12,4 @@ requests-toolbelt
 simplejson>=3.10.0
 scikit-image>=0.13.0
 statsmodels>=0.8.0
+simpleitk
```

Comment:
I don't remember if AllenSDK has an optional requirements file.  Depending on the size of this library, it seems like a candidate for an optional dependency.  But I don't feel strongly about it.